sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/PubsubToBigqueryIT.java

```diff
@@ -101,8 +97,14 @@ public void testSimpleInsert() throws Exception {
 
     BeamSqlRelUtils.toPCollection(pipeline, sqlEnv.parseQuery(insertStatement));
 
+    Supplier<Void> start = signal.waitForStart(Duration.standardMinutes(1));
```

Comment:
In other places this was used it seemed like 5 mins was the default.

```suggestion
    Supplier<Void> start = signal.waitForStart(Duration.standardMinutes(5));
```